packages/medusa-fulfillment-webshipper/src/services/webshipper-fulfillment.js

```diff
@@ -699,6 +699,13 @@ class WebshipperFulfillmentService extends AbstractFulfillmentService {
       webShipperItem.tarif_number = tarifNumber
     }
 
+    const weight = item?.variant?.weight || item?.variant?.product?.weight
```

Comment:
```suggestion
    const weight = item?.variant?.weight ?? item?.variant?.product?.weight
```

But if the weight is set to 0 by default somewhere, this might cause a bug. Let's check that.